model/vm_host.rb

```diff
@@ -188,6 +188,11 @@ def download_boot_image(image_name, version:, custom_url: nil)
     Strand.create_with_id(schedule: Time.now, prog: "DownloadBootImage", label: "start", stack: [{subject_id: id, image_name: image_name, custom_url: custom_url, version: version}])
   end
 
+  # Introduced for downloading a new firmware.
```

Comment:
A somewhat out of character comment. Is it trying to say that it is intended for console use?